code-challenges.js

```diff
@@ -28,18 +42,51 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// Pseudo code:
+console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+// Output: 9
+
+// Pseudo code: I used the .length to count the elements of each array and then added each variable to get the output of 9, representing the numbers of elements.
```

Comment:
Make sure you pay attention to details. The prompt asks you to combine the two arrays, then get the new length. However, in your code, you are getting the lengths of the 2 arrays and adding them together.